app/backend/lib/session.ts

```diff
@@ -24,7 +24,8 @@ const session = () => {
     secret: sessionSecret,
     store,
     resave: false,
-    saveUninitialized: true,
+    // testing saveUninitialized set to false so we can totally clear cookies on logout
```

Comment:
nit: this seems to be working fine so we can either change or remove this comment

Reply:
Done. Thanks!